dashboard/src/App.css

```
@@ -0,0 +1,963 @@
+}
+.explorer-toggle-btn:hover { color: var(--text); border-color: var(--border-hover); }
+.explorer-toggle-btn.active {
+  background: var(--surface);
+  backdrop-filter: blur(var(--glass-blur));
+  -webkit-backdrop-filter: blur(var(--glass-blur));
+  color: var(--text); border-color: var(--accent);
+  box-shadow: 0 0 12px var(--glow-accent);
+}
+
+/* Explorer Viewer */
+.explorer-viewer { }
```

Stylelint (17.3.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)